packages/utils/odsp-doclib-utils/src/test/odspErrorUtils.spec.ts

```diff
@@ -35,8 +50,10 @@ describe("OdspErrorUtils", () => {
 				400 /* statusCode */,
 				undefined /* retryAfterSeconds */,
 			);
+
+			assert(isIGenericNetworkError(networkError));
```

Comment:
Since this is the only place where we used the function, I think I'm leaning towards removing it, removing the new assert, and where we need to check the `statusCode` do `assert.equal((networkError as Partial<IGenericNetworkError>).statusCode, 400, "status code should be preserved");`